content/guides/use-case/jupyter/share.md

```diff
@@ -0,0 +1,143 @@
+This enables others to download, use, and build upon your work, fostering open
+science and collaborative development.
+
+## Prerequisites
+
+- You have installed the latest version of [Docker Desktop](../../../get-docker.md).
+- You have completed [Customize your JupyterLab environment](customize.md) and
+  saved a notebook to your local machine.
+
+## Add a notebook to a Docker image
+
+This example builds upon the image and notebook created in [Customize your JupyterLab environment](customize.md). You'll now add that saved notebook from your local machine to the image.
+
+Open the `Dockerfile` in and IDE or code editor and add a `COPY` instruction. The following is the contents of the `Dockerfile`.
+
+```dockerfile {hl_lines=5}
+# syntax=docker/dockerfile:1
+
+FROM jupyter/base-notebook
+RUN pip install --no-cache-dir matplotlib scikit-learn
+COPY ./*.ipynb /home/jovyan/work/
+
+```
+
+Use the `COPY` command in a Dockerfile to copy files and directories from the
+source file system (where the Dockerfile resides) into the filesystem of the
+newly built Docker image. The specific command `COPY ./*.ipynb
+/home/jovyan/work/` targets all IPython Notebook files (.ipynb) in the
+Dockerfile's directory, copying them into the `/home/jovyan/work/` directory
+inside the Docker container, making these notebooks available for use within the
+JupyterLab environment.
+
+Save the changes to your `Dockerfile`.
+
+Build your image using the same command from [Customize your JupyterLab environment](customize.md).
+
+```console
+$ docker build -t my-jupyter-image .
+```
+
+Finally, run your image without a bind mount to verify that the image contains your notebook.
+
+```console
+$ docker run --rm -p 8888:8888 my-jupyter-image start-notebook.py --NotebookApp.token='my-token'
+```
+
+> **Note**
+>
+> Specifying a bind mount in the `docker run` command for the
+> `/home/jovyan/work/` directory , or using Docker Compose with a bind mount
+> will overwrite the files copied into the image. For this example, make sure
+> you aren't using a bind mount.
+
+To access the container, in a web browser navigate to
+[localhost:8888/lab?token=my-token](http://localhost:8888/lab?token=my-token). You should find the notebook under the `work` directory.
+
+## Publish your image to Docker Hub
+
+Now that you have your notebook in an image, share it with the world. You'll need to create a repository on Docker Hub.
+
+### Create a repository on Docker Hub
+
+1. [Sign up](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade) or sign in to [Docker Hub](https://hub.docker.com).
+
+2. In Docker Hub, select **Repositories**, and then select the **Create
+   repository** button.
+
+3. For the repository name, use `my-jupyter-image`. Make sure the **Visibility**
+   is **Public**.
+
+4. Select **Create**.
+
+Now that you have a repository, you can push your local image to the remote repository.
```

Comment:
suggestion: we could just leave this out, you don't need to create a repository before pushing an image. If it doesn't exist, it will be created automatically.